app/views/node_info/_statistics.haml

```diff
@@ -0,0 +1,21 @@
+-#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
```

Comment:
@jaywink just for information, I changed the meta author in https://github.com/diaspora/diaspora/pull/6230/files#diff-b7186ba040870c69d2ae394fa317ea64R14 using "diaspora* contributors".